circle-mlir/infra/docker/u2004/Dockerfile

```diff
@@ -33,6 +33,11 @@ RUN python3 -m pip install onnx==${VER_ONNX} onnxruntime==${VER_ONNXRUNTIME}
 # Clean archives (to reduce image size)
 RUN apt-get clean -y
 
+# CMake 3.22
```

Comment:
onnx-mlir requires 3.20 or above. 3.22 is chosen as it's above 3.20, without any other reason.